showcase/google-cloud-jar-parent/pom.xml

```diff
@@ -0,0 +1,287 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-jar-parent</artifactId>
+  <groupId>com.google.cloud</groupId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>Google Cloud JAR Parent</name>
+  <description>
+    The parent for all client library JAR modules in the repository.
+  </description>
+
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-pom-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <relativePath>../google-cloud-pom-parent/pom.xml</relativePath>
```

Comment:
What's the benefit of having this pom and the parent pom? Is it just to mimicking the same behavior of [google-cloud-java](https://github.com/googleapis/google-cloud-java/blob/main/google-cloud-jar-parent/pom.xml)?

Reply:
Correct. I considered refactoring to reduce the complexity, but thought the similarity to google-cloud-java modules would be preferred.

Reply:
Personally I feel we don't need these two sub-sub modules as they are introducing more complexity without much benefit, but I would wait for @suztomo's decision. If we do decide to have these two modules, we need at least change the artifact ids and remove most of the dependencies below.

Reply:
I don't see the benefit of google-cloud-jar-parent or google-cloud-pom-parent here. In google-cloud-java repository, it helps to apply the same configuration to many modules.

In this repository, if you need some configuration in pom.xml apply directly to the module.

Reply:
- [ ] Condense showcase pom structure